worker-pools.yml

```diff
@@ -250,25 +250,113 @@ pools:
             - *scratch-disk
             - *scratch-disk
           machine_type: c2-standard-16
-  - pool_id: '{pool-group}/b-linux-amd'
+  - pool_id: '{pool-group}/b-linux{suffix}-amd'
     description: Worker for Firefox automation.
     owner: release+tc-workers@mozilla.com
     email_on_error: true
+    attributes:
+      # Used to populate scratch disks
+      scratch-disk-definition: *scratch-disk
```

Comment:
Maybe eventually we should move the `*scratch-disk` config into code, and just make every pool set `num_scratch_disks` or something like that.